src/altinn-app-frontend/src/utils/validation.test.ts

```diff
@@ -53,6 +57,63 @@ describe('utils > validation', () => {
       },
     };
 
+    mockComponent4 = {
```

Comment:
Good point, I also thought about that. I avoided that (larger) change, but I assumed they were being set each time because some tests change their values. Also, with `const`, I believe you could rely more on these mocks as types. Last week I tried changing a type from `string` to `keyof typeof mockLayout...`, but that won't work without them being `const` at least. Maybe I'll clean this up in a later PR!